.browserslistrc.json

```diff
@@ -0,0 +1,12 @@
+{
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+}
```

Comment:
Explore about `.browserslist` and add it correctly.

Reply:
done